src/main/java/appeng/crafting/v2/resolvers/EmitableItemResolver.java

```diff
@@ -73,7 +73,7 @@ public void fullRefund(CraftingContext context) {
         @Override
         public void populatePlan(IItemList<IAEItemStack> targetPlan) {
             if (fulfilled > 0 && request.stack instanceof IAEItemStack) {
-                targetPlan.addRequestable(request.stack.copy().setStackSize(fulfilled));
+                targetPlan.addRequestable(request.stack.copy().setCountRequestable(fulfilled));
```

Comment:
Why did you change this? This is how old ae2 code handled level emitters: <https://github.com/GTNewHorizons/Applied-Energistics-2-Unofficial/blob/8af0ae83317a0b807af088327a62f9b833f7d9e5/src/main/java/appeng/crafting/CraftingTreeNode.java#L187-L195>

Reply:
Since the items crafting by the transmitter will not be displayed in the crafting view, this change is made so that the items crafting by the transmitter will also be displayed
Before
![G0U{F94 %8}D25 L(ID4$KY](https://github.com/GTNewHorizons/Applied-Energistics-2-Unofficial/assets/30284813/03801091-7ae0-4e77-ba51-0418d6f61235)
After 
![41_TQFLEWJ_UXLV1V1%D QU](https://github.com/GTNewHorizons/Applied-Energistics-2-Unofficial/assets/30284813/6bf36593-b590-40d0-96fa-068b49268dce)

Reply:
I think this is not a display bug. The emitter does indeed submit a  crafting instead of submitting storage, and the storage set here will be ignored.